### PR TITLE
fix(Tabs): Removed isHelpAction prop from TabsAction

### LINKED ISSUE #8556: Tabs - remove isHelpAction from TabAction
re: https://github.com/patternfly/patternfly/pull/5330

The `.pf-m-help` class for a tab action was removed in core since it's beta and all it does is change the padding, and design requested the padding match all other actions. Assuming there is no other logic or reason for `isHelpAction` to exist (I didn't see anything), I think we can drop the prop and the `.pf-m-help` class it sets.

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8556

Also:
-  bumps the core version.
- Adds missing `TabAction` to documentation.

### CODE REVIEW

**Review by thatblindgeye — COMMENTED** [review]
Should `TabAction` have been added to the propComponents array in the Tabs.md file?

**Review by mcoker — APPROVED** [review]
Aside from @thatblindgeye's comment, lgtm!

File: packages/react-console/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@novnc/novnc": "^1.3.0",
-    "@patternfly/patternfly": "4.222.4",
+    "@patternfly/patternfly": "4.223.2",
```
Comment (mcoker) [inline packages/react-console/package.json:36]: can we bump to [4.224.0](https://www.npmjs.com/package/@patternfly/patternfly/v/4.224.0) in this PR? hopefully that's the last core version for this release, it includes support for this v4 issue https://github.com/patternfly/patternfly-react/issues/8434

**Comment by tlabaj** [comment]
> Should `TabAction` have been added to the propComponents array in the Tabs.md file?

I forget to push the change. It should be there now.

**Review by thatblindgeye — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**